scripts/globals/magic_utils/spell_damage.lua

```diff
@@ -213,11 +213,21 @@ xi.magic_utils.spell_damage.calculateSDT = function(caster, target, spell, spell
         SDTMod = target:getMod(xi.magic.specificDmgTakenMod[spellElement])
 
         -- TODO: Ths conversion right here needs to be updated once work on SDT modifiers is finished.
+        -- Current behaviour goes like this:
+        -- In the database, the modifiers as set as the numbers we actually want.
+        -- A 1 = same damage/no change.
+        -- A 0.5 = Half damage.
+
+        -- THEN in C++ land, in zoneutils.cpp we CONVERT this values with the following equation:
+        -- Mod::FIRE_SDT, (int16)((Sql_GetFloatData(SqlInstanceHandle, 41) - 1) * -100));
+        -- And we populate the modifiers with said NEW number. We wanted the old one though, so we need to convert it back here.
         SDT = (SDTMod / -100) + 1
     end
 
     -- SDT (Species/Specific Damage Taken) is a stat/mod present in mobs and players that applies a % to specific damage types.
     -- Think of it as an extension (or the actual base) of elemental resistances in past FF games.
```

Comment:
I know, pre-existing line but: we really do not want to think of using the word "resistances" because thats exactly how people get confused about what it does. anything "DT" is Damage Taken meaning reduction and not resistance which is randomish. DT is always.

Reply:
For sure, I'll edit it